
// 1.6 Array loops

console.log("Excercise 7.5");

// Gestió d'errors amb async/await: Modifica la funció de l'exercici 4 per a que capturi qualsevol possible error utilitzant un bloc try/catch.
